Source/Core/Credit.js

```diff
@@ -36,7 +36,7 @@ function Credit(html, showOnScreen) {
 
   showOnScreen = defaultValue(showOnScreen, false);
 
-  // Credits are immutable so generate an id to use to optimize equal()
+  // Credits are mostly immutable, so generate an id to use to optimize equal()
```

Comment:
I think it's fair to say that they are immutable still; at least the credit string which is used to generate the id is immutable